// react-native-navbar/NavbarButton.js
import React from 'react-native';
const {
  Text,
  TouchableOpacity,
  Component,
  View,
  PropTypes,
} = React;
import styles from './styles';
var { Icon, } = require('react-native-item-checkbox');

export default class NavbarButton extends Component {
  render() {
    const { style, tintColor, margin, title, handler, type } = this.props;

    if(type == 'flagIcon') {
      return (
        <TouchableOpacity onPress={handler}>
          <View style={style}>
              <Icon
                name='ion|flag'
                size={ 20 }
                color='#0066CD'
                style={[{ backgroundColor: 'white', width: 20, height: 20, }, styles.navBarButtonText]} />
          </View>
        </TouchableOpacity>
      );
    } else {
      return (
        <TouchableOpacity onPress={handler}>
          <View style={style}>
            <Text style={[styles.navBarButtonText, { color: tintColor, }, ]}>{title}</Text>
          </View>
        </TouchableOpacity>
      );
    }
  }

  static propTypes = {
    style: PropTypes.oneOfType([
      PropTypes.object,
      PropTypes.array,
    ]),
    tintColor: PropTypes.string,
    title: PropTypes.string,
    handler: PropTypes.func,
  };

  static defaultProps = {
    style: {},
    title: '',
    tintColor: '#0066CD',
    onPress: () => ({}),
  };
}
